### PR TITLE
make demo contenteditable

### PR DESCRIPTION
**Pull request by Haroenv**
since this is only CSS, contenteditable is fine to use here I think

### CODE REVIEW

**Comment by yershalom** [comment]
Hi,
Can you please explain?
Not sure I understand why...

**Comment by Haroenv** [comment]
This way you can play around with the text and make your own wordart in the demo

File: example/src/App.js [hunk 1]
```diff
@@ -3,18 +3,23 @@ import React, { Component } from 'react'
 import WordArt from 'react-wordart'
 
 export default class App extends Component {
-  render () {
+  render() {
     return (
-      <div>
-        <a className='github-button'
+      <div contenteditable>
```
Comment (yershalom) [inline example/src/App.js]: Please change to `contentEditable `

Comment (Haroenv) [reply example/src/App.js]:
```suggestion
      <div contentEditable>
```

**Pull request merged**